package.json

```diff
@@ -2,7 +2,7 @@
     "name": "ui-options-chrome",
     "version": "0.1.0",
     "description": "User Interface Options Plus (UIO+) allows you to customize websites to match your own personal needs and preferences.",
-    "license": "(BSD-3-Clause OR ECL-2.0)",
+    "license": "BSD-3-Clause",
     "repository": {
         "type": "git",
         "url": "git://github.com/GPII/gpii-chrome-extension.git"
```

Comment:
Probably as part of this pull the repository URL should be fixed up

Reply:
Also the homepage URL and also the bugs URL, given it seems we are planning to move to github issues - I will merge this set of pulls in in any case since it is not completely clear which one these adjustments should fall under